plugins/ltac2/tac2stdlib.ml

```diff
@@ -397,6 +397,11 @@ let () =
     (bool @-> list rewriting @-> clause @-> option (thunk unit) @-> tac unit)
     Tac2tactics.rewrite
 
+let () =
+  define "tac_setoid_rewrite"
+    (bool @-> uthaw constr_with_bindings @--> occurrences @-> option ident @-> tac unit)
```

Comment:
```suggestion
    (orientation @-> uthaw constr_with_bindings @--> occurrences @-> option ident @-> tac unit)
```
?

Reply:
I don't think it's worth it when the ocaml orientation is just bool

Reply:
@JasonGross to be able to work, the change you suggest is a bit more involved as it would also need the addition of a `to_orientation` function (converting the Ltac2 `orientation` value back to OCaml), very similar to `to_bool`. This might be overkill (as @SkySkimmer says), so let me know what I should do.

Reply:
I second @SkySkimmer

Reply:
Sounds good